1-exercises/B-while-loop/exercise.js

```diff
@@ -6,7 +6,14 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+    let i = 0;
+    while (i < n*2){
+        if (n === 0) break;
+        if (i % 2 === 0) {
+        console.log(i);
```

Comment:
Maybe I'm nitpicking, but the output should be comma delimited. Console.log on each run would output them in a new line each run.